app/routes.py

```diff
@@ -1,2 +1,325 @@
-from flask import Blueprint
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from flask import request, Blueprint, make_response, jsonify
+import os
+import requests
+from datetime import datetime, timedelta
+
+
+customer_bp = Blueprint("customers", __name__, url_prefix="/customers")
+video_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rental_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+@customer_bp.route("", methods=["POST"], strict_slashes=False)
+def create_new_customer():
+    request_body = request.get_json()
+
+    if not request_body:
+            return make_response({"details": "Invalid data"}, 400)
+    
+    
+    if not request_body.get("name"): 
+        return make_response({"details": "Please enter a value for Name"}, 400)
+    elif not request_body.get("postal_code"):
+        return make_response({"details": "Please enter a value for postal code"}, 400)
+    elif not request_body.get("phone"):
+        return make_response({"details": "Please enter a value for phone"}, 400)
+
+    new_customer = Customer.from_json(request_body)
+
+    db.session.add(new_customer)
+    db.session.commit()
+
+    return make_response({"id": f"{new_customer.id}"}, 201)
```

Comment:
_minor_:  This should be a number 

```suggestion
    return make_response({ "id": new_customer.id }, 201)
```